astropy/stats/tests/test_funcs.py

```diff
@@ -9,6 +9,8 @@
 from numpy.testing import assert_equal
 from numpy.testing.utils import assert_allclose
 
+from scipy.stats import spearmanr
```

Comment:
scipy is an optional dependency at the moment, so this import should be done in your test function, which should be decorated with ``@pytest.mark.skipif('not HAS_SCIPY')``. There are a few examples below of how to do that.